app/src/main/java/com/sebastianvm/mango/database/models/TaxEntity.kt

```diff
@@ -5,7 +5,9 @@ import com.sebastianvm.mango.model.Deduction
 import com.sebastianvm.mango.model.Jurisdiction
 import com.sebastianvm.mango.model.Tax
 import com.sebastianvm.mango.model.TaxType
+import kotlinx.serialization.Serializable
 
+@Serializable
```

Comment:
How do you feel about making the entity serializable? Maybe this is why the models should also be data classes?